apps/src/applab/applab.js

```diff
@@ -697,6 +697,7 @@ Applab.init = function(config) {
   }
 
   if (experiments.isEnabled('student-libraries') && librariesExist) {
+    config.dropletConfig.additionalPredefValues.push(level.libraries[0].name);
```

Comment:
What if the user imports multiple libraries? (Even if not implemented yet, we may want to think ahead for that case here so we don't have to go back and update later.)

Reply:
Good point. Fixed!